.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,6 +6,16 @@ labels: kind/bug
 assignees: ''
 ---
 
+<!--
+In order to help us process this issue faster, please provide as much detail as possible.
+
+Before creating a new request for help, please search existing issue to see if you find a similar one.
```

Comment:
```suggestion
Before creating a new request for help, please search existing issues to see if you find a similar one.
```

Reply:
you should probs update that in eksctl too :D